src/cli/global/list.rs

```diff
@@ -86,22 +76,58 @@ pub async fn execute(_args: Args) -> miette::Result<()> {
                 .into_iter()
                 .collect();
 
+        let version = prefix_package
+            .repodata_record
+            .package_record
+            .version
+            .to_string();
         package_info.push(InstalledPackageInfo {
             name: package_name,
             binaries,
+            version,
         });
     }
 
     if package_info.is_empty() {
         print_no_packages_found_message();
     } else {
-        eprintln!(
-            "Globally installed binary packages:\n{}",
-            package_info
-                .into_iter()
-                .map(|pkg| pkg.to_string())
-                .join("\n")
-        );
+        let path = home_path()?;
+        let len = package_info.len();
+        let mut message = String::new();
+        for (idx, pkgi) in package_info.into_iter().enumerate() {
+            let last = (idx + 1) == len;
+            let no_binary = pkgi.binaries.is_empty();
+
+            if last {
+                message.push_str("└──");
+            } else {
+                message.push_str("├──");
+            }
+
+            message.push_str(&format!(
+                " {} {}",
+                console::style(&pkgi.name.as_source()).bold(),
+                console::style(&pkgi.version).bright().black()
+            ));
+
+            if !no_binary {
+                let p = if last { " " } else { "|" };
+                message.push_str(&format!(
+                    "\n{}   └─ bin: {}",
```

Comment:
What do you think about naming this `exec` for executable instead of `bin` as it might come across as a folder structure where it kinda is but not exactly.

Reply:
`exec` sounds better indeed.